crawl-ref/source/timed-effects.cc

```diff
@@ -1417,28 +1414,31 @@ void update_level(int elapsedTime)
     delete_all_clouds();
 }
 
+/**
+ * Update the monster upon the player's return
+ *
+ * @param mon   The monster to update.
+ * @param turns How many turns the monster has been away from the player.
```

Comment:
Might be a good idea to explicitly state that it's turns and not auts. I have trouble sometimes when I just see "turns" because sometimes we say turns but mean auts. @neilmoore may disagree with me though.